apps/pie-docs/src/assets/styles/components/_toggle.scss

```diff
@@ -1,30 +1,27 @@
 @use 'sass:color';
 @use '../fozzie-config' as f;
 
-$toggle-bg: f.$color-container-strong;
+$toggle-bg: var(--dt-color-container-strong);
 $toggle-radius: f.$radius-rounded-e;
 $toggle-padding: 2px;
 $toggle-item-height: 44px;
 $toggle-item-fw: f.$font-weight-regular;
 $toggle-item-fw--checked: f.$font-weight-bold;
 $toggle-item-padding-y: 10px;
 $toggle-item-padding-x: var(--dt-spacing-e);
-$toggle-item-bg: f.$color-container-strong;
+$toggle-item-bg: var(--dt-color-container-strong);
 
 // This is really annoying, but it's the only way to get darkness to work with Custom Props
```

Comment:
Looks like this comment can be deleted now?

Reply:
will remove it in my next PR 🤝

Reply:
never mind I need to update the changeset anyway